Program.cs

```diff
@@ -6,17 +6,36 @@
 
 namespace CurrentValueCalculator_HW2
 {
-    enum TransactionType
-    {
-        Buy,
-        Sell
+    enum TransactionType {
+        Buy, Sell
     }
+    class Program {
+        static void Main(string[] args) {
+            Console.WriteLine("Input the nominal of the trade:");
+            string userInput = Console.ReadLine();
+            int nominal = int.Parse(userInput ?? "0");
 
-    class Program
-    {
-        static void Main(string[] args)
-        {
+            Console.WriteLine("Input the trade price of the deal:");
+            userInput = Console.ReadLine();
+            double tradePrice = double.Parse(userInput ?? "0");
```

Comment:
you can remove this condition - `userInput ?? "0"`; 
If user won't input anything in the console, Console.ReadLine would return `string.Empty`;
So a better way to do such condition would be 
`tradePrice = string.IsNullOrEmpty(userInput) ? 0 : double.Parse(userInput)`

Reply:
and please consider changing all occurences of `userInput ?? "0"`